src/lib/cli.rs

```diff
@@ -144,10 +144,25 @@ fn endpoints_parser(entry: &str) -> Result<Arc<dyn drivers::Driver>, String> {
 /// Constructs our manager, Should be done inside main
 #[instrument(level = "debug")]
 pub fn init() {
-    init_with(Args::parse());
+    let expanded_args = std::env::args()
+        .map(|arg| {
+            // Fallback to the original if it fails to expand
+            shellexpand::env(&arg.clone())
+                .inspect_err(|_| {
+                    warn!("Failed expanding arg: {arg:?}, using the non-expanded instead.")
+                })
+                .unwrap_or_else(|_| arg.into())
+                .into_owned()
+        })
+        .collect::<Vec<String>>();
+
+    let reparsed_expanded_args = Args::parse_from(expanded_args);
+
+    init_with(reparsed_expanded_args);
 }
 
 /// Constructs our manager, Should be done inside main
+/// Note: differently from init(), this doesn't expand env variables
```

Comment:
Just an additional comment here, this would be very hard to do, as we don't have a simple and maintainable way to obtain a command-line string from Args and re-parse it after expanding.